### PR TITLE
JSDK-2699/JSDK-2713

### PR DESCRIPTION
**Pull request by PikaJoyce**
This is a PR for both JSDK-2699 as well as JSDK- 2713

Added documentation updates for how to run unit and integration tests (JSDK-2699).
Updated DominantSpeakerPriority documentation (JSDK-2713).

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -166,6 +166,23 @@ npm run build
...
+
+```
+npm run test:unit
+```
+
+Run integration tests with 
+
+```
+ACCOUNT_SID=<Your account sid>
+API_KEY_SID=<Your api key sid>
```
Comment (makarandp0) [inline README.md]: should mention that `TOPOLOGY` need to be set as well.

File: lib/connect.js [hunk 2]
```diff
@@ -443,6 +443,7 @@ function connect(token, options) {
  *   lower {@link Track.Priority} will be subscribed to with the {@link Track.Priority} specified here.
  *   This has no effect on {@link RemoteVideoTrack}s published with higher {@link Track.Priority}, which will
  *   still be subscribed to with with the same {@link Track.Priority}. If not specified, this defaults to "standard".
+ *   This parameter only applies to a Group Room Participant when ConnectOptions.dominantSpeaker is set to true.
```
Comment (manjeshbhargav) [inline lib/connect.js]:
```suggestion
 *   This parameter only applies to a Group Room Participant when {@link ConnectOptions}.dominantSpeaker is set to true.
```

File: README.md [hunk 3]
```diff
@@ -166,6 +166,36 @@ npm run build
...
+```
+npm run test:unit
+```
+
+Run integration tests with 
+
+```
+ACCOUNT_SID=<Your account sid> \
+API_KEY_SID=<Your api key sid> \
+API_KEY_SECRET=<Your api key secret>
```
Comment (makarandp0) [inline README.md]:
```suggestion
API_KEY_SECRET=<Your api key secret>  \
```

File: README.md [hunk 4]
```diff
@@ -166,6 +166,36 @@ npm run build
...
+npm run test:unit
+```
+
+Run integration tests with 
+
+```
+ACCOUNT_SID=<Your account sid> \
+API_KEY_SID=<Your api key sid> \
+API_KEY_SECRET=<Your api key secret> 
+BROWSER=<Browser you'd like to use>
```
Comment (makarandp0) [inline README.md]:
```suggestion
BROWSER=<Browser you'd like to use> \
```

**Pull request merged**